src/DashboardUI/src/pages/admin/AdminOrganizationUsersPage.tsx

```diff
@@ -61,23 +67,86 @@ export function AdminOrganizationsUsersPage() {
     }
   };
 
+  const openRemoveUserModal = (userId: string) => {
+    setShowRemoveUserModal(!showRemoveUserModal);
```

Comment:
Probably doesn't matter, but if the method is called open I figure we should always pass `true`
```suggestion
    setShowRemoveUserModal(true);
```